config/sync/core.entity_form_display.media.google_form.default.yml

```diff
@@ -0,0 +1,60 @@
+uuid: 84b810f0-d451-4792-b08e-09e64af9d278
```

Comment:
Why not have this config in stanford_media?

Reply:
we could or we dont need to. i have no objections either way. it just comes down to the question: Should new sites that are just starting (not the ones in on this repo) to use stanford media get the media type?

Reply:
That is where we are managing the other media types. I think it makes sense to me.

Reply:
Please move it along

Reply:
https://github.com/SU-SWS/stanford_media/pull/52